ODIN_II/SRC/netlist_utils.cpp

```diff
@@ -239,20 +239,31 @@ npin_t* free_npin(npin_t* to_free) {
 	return (npin_t*)vtr::free(to_free);
 }
 
+/*-------------------------------------------------------------------------
+ * (function: copy_npin)
+ * 	Copies a pin
+ *-----------------------------------------------------------------------*/
+npin_t *copy_npin(npin_t *copy_pin)
```

Comment:
make this function static and remove it from the .h file since it is a helper function, we don't want it used by itself, should be usd via copy input or copy output